_sass/members.scss

```diff
@@ -0,0 +1,228 @@
+	transition: transform .7s cubic-bezier(0.4, 0.2, 0.2, 1), -webkit-transform .7s cubic-bezier(0.4, 0.2, 0.2, 1);
+	-webkit-backface-visibility: hidden;
+	backface-visibility: hidden;
+	text-align: center;
+	min-height: 280px;
+	height: auto;
+	border-radius: 10px;
+	color: #fff;
+	font-size: 1.5rem;
+	&:after {
+		position: absolute;
+		top: 0;
+		left: 0;
+		z-index: 1;
+		width: 100%;
+		height: 100%;
+		content: '';
+		display: block;
+		opacity: 0;
+		background-color: #000;
+		-webkit-backface-visibility: hidden;
+		backface-visibility: hidden;
+		border-radius: 10px;
+	}
+	.inner {
+		p {
+			font-size: 1.6rem;
+			margin-bottom: 2rem;
+			position: relative;
+			&:after {
+				content: '';
+				width: 4rem;
+				height: 2px;
+				position: absolute;
+				background: #C6D4DF;
+				display: block;
+				left: 0;
+				right: 0;
+				margin: 0 auto;
+				bottom: -.75rem;
+			}
+		}
+		span {
+			color: rgba(255,255,255,0.7);
+			font-family: 'Montserrat';
+			font-weight: 300;
+		}
+	}
+}
+.back {
+	background-size: cover;
+    box-shadow: 0 4px 8px 0 rgba(0,0,0,0.25);
+	border-radius: 10px;
+	background-position: center;
+	-webkit-transition: -webkit-transform .7s cubic-bezier(0.4, 0.2, 0.2, 1);
+	transition: -webkit-transform .7s cubic-bezier(0.4, 0.2, 0.2, 1);
+	-o-transition: transform .7s cubic-bezier(0.4, 0.2, 0.2, 1);
+	transition: transform .7s cubic-bezier(0.4, 0.2, 0.2, 1);
+	transition: transform .7s cubic-bezier(0.4, 0.2, 0.2, 1), -webkit-transform .7s cubic-bezier(0.4, 0.2, 0.2, 1);
+	-webkit-backface-visibility: hidden;
+	backface-visibility: hidden;
+	text-align: center;
+	min-height: 280px;
+	height: auto;
+	border-radius: 10px;
+	color: #fff;
+	font-size: 1.5rem;
+	background: #cedce7;
+	background: -webkit-linear-gradient(45deg, #292929 0%,#000000 100%);
+	background: -o-linear-gradient(45deg, #292929 0%,#000000 100%);
+	background: linear-gradient(45deg, #292929 0%,#000000 100%);
+	position: absolute;
+	top: 0;
+	left: 0;
+    width: 100%;
+    
+}
+
+.back a{
+    color:white;
+}
+.inner {
+	-webkit-transform: translateY(-50%) translateZ(60px) scale(0.94);
+	transform: translateY(-50%) translateZ(60px) scale(0.94);
+	top: 50%;
+	position: absolute;
+	left: 0;
+	width: 100%;
+	padding: 2rem;
+	-webkit-box-sizing: border-box;
+	box-sizing: border-box;
+	outline: 1px solid transparent;
+	-webkit-perspective: inherit;
+	perspective: inherit;
+	z-index: 2;
+}
+
+
+.megamind-name {
+    position: absolute;
+    bottom: 10px;
+    left: 10px;
+    background: rgba(0, 0, 0, 0.75);
+    padding: 4px 8px;
+    color: white;
+    margin: 0;
+    font: 20px Sans-Serif;
+  }
+
+
+.member-description{
+    font-size: 1rem;
+}
+
+#front-view{
+    top:90%;
+}
+@media screen and (max-width: 64rem) {
+	.col {
+		width: calc(33.333333% - 2rem);
+	}
+}
+@media screen and (max-width: 48rem) {
+	.col {
+		width: calc(50% - 2rem);
+	}
+}
+@media screen and (max-width: 32rem) {
+	.col {
+		width: 100%;
+		margin: 0 0 2rem 0;
+	}
+}
+
```

Comment:
remove the additional lines, the reason is that the number of lines changed includes the additional gaps